.env.base

```diff
@@ -167,8 +167,6 @@ REACT_APP_SNAP_ID=npm:@shapeshiftoss/metamask-snaps
 REACT_APP_SNAP_VERSION=1.0.9
 # REACT_APP_SNAP_ID=local:http://localhost:9000
 
-REACT_APP_EXPERIMENTAL_MM_SNAPPY_FINGERS=true
```

Comment:
Was true for a long time, removed hwallet-metamask in companion hdwallet PR and now only using multichain hdwallet